homeassistant/components/google/quality_scale.yaml

```diff
@@ -0,0 +1,87 @@
+rules:
+  # Bronze
+  config-flow:
+    status: done
+    comment: Integration uses an OAuth config flow.
+  brands: done
+  dependency-transparency: done
```

Comment:
https://github.com/googleapis/oauth2client is not built and published in the CI. It's archived so I am not sure if this is easy to work around

Reply:
What is interesting is that the rule doesn't apply to dependencies. (It kind of incentivizes putting a wrapper in the client library to just abstract it by one layer, which likely is not what is intended here). But maybe i'll just rewrite it to be asyncio.

Reply:
I noticed this because `nest` does not directly use the python google pubsub client library (which is published the same way, internally using a CI pipeline inside google and not using github), but the nest subscriber library wraps it, and it does not violate the rule.

Reply:
At this point its also a bit of due dilligence and I am trying to see the repo of the library to also see their dependencies. But its a good point nevertheless

Reply:
I've marked as `todo` with context.